src/app/components/pages/IsaacCompetition.tsx

```diff
@@ -1,24 +1,74 @@
 
 export const IsaacCompetition = () => {
   useEffect(() => {
     document.title = "Isaac " + SITE_SUBJECT_TITLE;
   }, []);
 
   return (
-    <Container>
+    <>
       <TitleAndBreadcrumb currentPageTitle="Isaac Competition" breadcrumbTitleOverride="Isaac Competition" />
 
       <div id="section1">
         <h1>Section 1</h1>
       </div>
 
-      <div id="section2">
-        <h1>Section 2</h1>
-      </div>
-    </Container>
+      <section id="internetOfEverything" className="event-section">
+        <div className="event-section-background-img">
+          <div className="container">
+            <div className="row ioe-row pt-4 pb-4">
+              <div className="ioe-custom-width ioe-col p-4 rounded">
+                <h3 className="ioe-title">What is the Internet of Everything?</h3>
+                <p className="ioe-text pt-3">
+                  Imagine a world where everything around you is connected to the internet. Not just your phone or
+                  computer, but your watch, your shoes, your fridge, even your toothbrush! This is what we call the
+                  Internet of Everything.
+                </p>
+                <p className="ioe-text pt-3">
+                  The IoE is like a giant invisible web that connects all things, people, data, and processes. It’s like
+                  a big team where everyone and everything works together, sharing information and making decisions.
+                </p>
+                <p className="ioe-text pt-3">
+                  In this competition, students will create a concept or prototype for an IoE product and showcase it
+                  through a video submission.
+                </p>
+              </div>
+              <div className="ioe-custom-width ioe-col p-4 rounded">
+                <h3 className="ioe-title">Real-life examples of IoE include:</h3>
+                <ul className="ioe-text pt-3">
+                  <li>
+                    smart homes: imagine your alarm clock wakes you up and then automatically tells your coffee maker to
+                    start brewing coffee
+                  </li>
+                  <li>
+                    wearable devices: devices like smartwatches can monitor your health, track your location if you’re
+                    lost, and even let you make phone calls or send messages
+                  </li>
+                  <li>
+                    smart cities: streetlights that turn off when it is light outside, or traffic lights that adjust
+                    based on road conditions
+                  </li>
+                  <li>
+                    connected cars: cars can communicate with each other to avoid accidents. They can also tell you when
+                    they need repairs or even call for help if there’s an emergency
+                  </li>
+                </ul>
+              </div>
+              <div className="ioe-comment-width ioe-comment-content">
```

Comment:
could the one for the homepage not be reused?

Reply:
Sadly no, I tried that first, however the font sizes are different and some of the styling is different so I had to make a new one.

Reply:
we should get clarification on that then, they should be the same for consistency.